src/App.css

```diff
@@ -5,16 +5,25 @@
 
 body {
```

Comment:
- [ ] Your app is not styled correctly, kindly consider making the app look exactly like the original design

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220723_154852](https://user-images.githubusercontent.com/25227022/180610397-d4a5fc95-c0a2-41a0-a9b8-d28a6d5724e3.png)|![bookstore (1)](https://user-images.githubusercontent.com/25227022/180610464-c7701897-b141-4a71-a57e-a26d7182c641.png)|